Digipost.Api.Client.DataTypes/Appointment/Appointment.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Xml;
+
+namespace Digipost.Api.Client.DataTypes.Appointment
+{
+    public class Appointment : IDataType
+    {
+        public Appointment(DateTime startTime)
+        {
+            StartTime = startTime;
+        }
+
+        public DateTime StartTime { get; set; }
+
+        /// <summary>
+        ///     Default value 30 minutes after <see cref="StartTime" />.
```

Comment:
En kommentar til en slarv som har gjort dette før deg (les: meg): Bør vi kanskje ha punktum i slutten av alle disse, bare sånn fordi? Det er jo ikke alle kommentarene som er setninger, så da er det jo på en måte rett å la være?

Reply:
Mjæ, det mest korrekte er vel å ha punktum på slutten av alle. Og så _ikke_ ha det i lister og sånt. Men jevnt over er det vel setninger over alt? Eller? Tror ikke det spiller så mye rolle for meg om det er punktum eller ikke, men enig i at det hadde vært en fin ting om det var likt overalt.